cookbooks/aws-parallelcluster-environment/files/redhat-8.network_interfaces/configure_nw_interface.sh

```diff
@@ -23,9 +24,10 @@ then
 fi
 
 con_name="System ${DEVICE_NAME}"
-route_table="100${DEVICE_NUMBER}"
-priority="100${DEVICE_NUMBER}"
-metric="100${DEVICE_NUMBER}"
+SUFFIX=$(printf "%03d" $NETWORK_CARD_INDEX)$(printf "%03d" $DEVICE_NUMBER)
```

Comment:
To be consistent with existing code we should use lowercase variable names for internal variables, so we should have `suffix` rather than `SUFFIX`.